data/killswitch.appdata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2021 Pong Loong Yeat <pongloongyeat@gmail.com> -->
+<component type="desktop">
+  <id>com.github.pongloongyeat.killswitch</id>
+  <metadata_license>CC0</metadata_license>
+  <name>KillSwitch</name>
+  <summary>One app to kill them all</summary>
+  <description>
+    <p>Kill all applications or desktop components with just a single click.</p>
+  </description>
+</component>
```

Comment:
Could you separate publishing requirements from this PR and put them in a separate PR? i.e. ignore the appdata.xml file for now.

Reply:
yeah, sure 👍🏼